pyproject.toml

```diff
@@ -1,110 +1,110 @@
 [project]
 name = "astropy"
-dynamic = [
-    "version"
-]
 description = "Astronomy and astrophysics core library"
 readme = "README.rst"
+keywords = [
```

Comment:
The alphabetization has removed meaning here. `astropy` is not first and foremost about `ascii` (not sure why that was put on there, though I guess table reading...).